pwiz_tools/Skyline/Controls/Graphs/AreaCVToolbar.cs

```diff
@@ -26,7 +26,7 @@
 
 namespace pwiz.Skyline.Controls.Graphs
 {
-    public sealed partial class AreaCVToolbar : GraphSummaryToolbar //UserControl // for designer
+    public sealed partial class AreaCVToolbar : GraphSummaryToolbar //GraphSummaryToolbar UserControl // for designer

```

Comment:
This seems like left-over debugging code. I think I fixed this to no longer be necessary, switching to UserControl for the designer.

Reply:
It did not work for me, I had to use the UserControl method.